tools/@aws-cdk/pkglint/lib/rules.ts

```diff
@@ -118,8 +118,10 @@ export class RepositoryCorrect extends ValidationRule {
   public validate(pkg: PackageJson): void {
     expectJSON(this.name, pkg, 'repository.type', 'git');
     expectJSON(this.name, pkg, 'repository.url', 'https://github.com/aws/aws-cdk.git');
-    const pkgDir = path.relative(monoRepoRoot(), pkg.packageRoot);
-    expectJSON(this.name, pkg, 'repository.directory', pkgDir);
+    const osSpecificRepositoryDirectory = path.relative(monoRepoRoot(), pkg.packageRoot);
+    // Enforcing '/' as separator for builds to work in Windows.
+    const posixStyleRepositoryDirectory = osSpecificRepositoryDirectory.split(path.sep).join('/');
+    expectJSON(this.name, pkg, 'repository.directory', posixStyleRepositoryDirectory);
```

Comment:
keep the terse names.

```suggestion
    const pkgDir = path.relative(monoRepoRoot(), pkg.packageRoot);
    // Enforcing '/' as separator for builds to work in Windows.
    const osPkgDir = pkgDir.split(path.sep).join('/');
    expectJSON(this.name, pkg, 'repository.directory', osPkgDir);
```